templates/publisher/account-snaps.html

```diff
@@ -0,0 +1,166 @@
+                  {% if snaps[snap].icon_url %}
+                    <img src="{{ snaps[snap].icon_url }}" class="p-snap-list__image" />
+                  {% else %}
+                    <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" class="p-snap-list__image" />
+                  {% endif %}
+                  <span class="p-snap-list__name u-no-margin--top">
+                    {{snap}}
+                  </span>
+                </a>
+              </td>
+              <td>
+                {% if snaps[snap].private %}
+                  Private
+                {% else %}
+                  Public
+                {% endif %}
+              </td>
+              <td>
+                {% if snaps[snap].publisher.username == current_user %}
+                  You
+                {% else %}
+                  {{ snaps[snap].publisher['display-name'] }}
+                {% endif %}
+              </td>
+              {% if snaps[snap].latest_revisions[0].status == 'Published' %}
+                <td>{{ snaps[snap].latest_revisions[0].channels[0] }}</td>
+              {% else %}}
+                <td>Not released</td>
+              {% endif %}
+              <td>{{ snaps[snap].latest_revisions[0].version }}</td>
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+    {% if snaps|length == 1 %}
+      {% for snap in snaps %}
+        <div class="p-strip--light is-shallow">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--five">Congratulations!</h3>
+            </div>
+            <div class="col-6">
+              <p>
+                You've just released a snap!<br />
+                {% if not snaps[snap].private %}
+                  This snap is public &mdash; anyone can find and install it.
+                {% else %}
+                Currently your app is set to private.
+                {% endif %}
+              </p>
+              <p>Things to do next:</p>
+            </div>
+          </div>
+          <div class="row u-no-margin--top">
+            <hr />
+          </div>
+          <div class="row u-no-margin--top">
+            <div class="col-3">
+              <h4>Market</h4>
+            </div>
+            <div class="col-5">
+              <p>
+                Want to improve the listings in stores?
+                You can add an icon and screenshots.
+              </p>
+              <p>
+                <a href="/account/snaps/{{ snap }}/market" class="p-button--neutral">Go to market listing</a>
+              </p>
+            </div>
+          </div>
+        </div>
+      {% endfor %}
+    {% endif %}
+  </div>
+{% endif %}
+{% if registered_snaps %}
+  <div class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-12">
+        <h4 class="u-float--left">My registered snap names</h4>
```

Comment:
Is `u-float--left` needed here? Is there anything else supposed to be here in this row for the float to be needed?

Reply:
Good catch - in the original design there was a link to the right for registering a name, which has been removed. Fixed.